### PR TITLE
Implement expiring api access tokens

### PR DESCRIPTION
**Pull request by jackleland**
Builds on permissions changes in #29. 

Adds additional setting (in settings.py) and authentication model which invalidates tokens created longer ago than the set expiry time. The auth-token endpoint can be used to get a refreshed token (using a valid username and password). Utilised only existing models so no migrations required.  

This addresses the problem of an access-token getting exposed in some manner and then forever compromising the api interface.

### CODE REVIEW

**Comment by jackleland** [comment]
This requires additional, fine grained control over the expiry of tokens for different users

**Comment by jackleland** [comment]
See https://github.com/HeloiseS/atlasapiclient/issues/4

**Comment by jackleland** [comment]
Fine-grained control over authentication, specifically different token expiries for different groups, is now implemented. This required a database change and so requires migrations for the feature to work as expected - I'm still working on a neat solution but this can be achieved right now through Django migrations if you run (in the container or otherwise, before pull these changes) a fake migration with 
``` bash
python manage.py makemigrations atlas
python manage.py migrate atlas --fake 
```

and then pull these changes and run the `makemigrations atlas` and `migrate atlas` again. This should then correctly make the required table `auth_group_profile`. 

This is not a neat solution right now and probably requires either a manual migration to be written or an updated init.sql file to be distributed.

File: psat_server_web/atlas/atlas/models.py [hunk 1]
```diff
@@ -1048,7 +1082,7 @@ class TcsGravityAlerts(models.Model):
     creator = models.CharField(max_length=30, blank=True, null=True)
     group = models.CharField(max_length=100, blank=True, null=True)
     pipeline = models.CharField(max_length=100, blank=True, null=True)
-    map_iteration = models.CharField(max_length=100, blank=True, null=True)
+    map_iteration = models.CharField(max_length=100, unique=True, blank=True, null=True)
```
Comment (genghisken) [inline psat_server_web/atlas/atlas/models.py:1085]: This is always requested by Django, but I don't believe it needs to be actually done. Ken to find out why!

File: psat_server_web/atlas/atlas/settings.py [hunk 2]
```diff
@@ -25,9 +25,9 @@
 SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (genghisken) [inline psat_server_web/atlas/atlas/settings.py:28]: Set back to False always for deployment reasons

File: psat_server_web/atlas/atlas/settings.py [hunk 3]
```diff
@@ -25,9 +25,9 @@
...
 
-TEMPLATE_DEBUG = False
+TEMPLATE_DEBUG = True
```
Comment (genghisken) [inline psat_server_web/atlas/atlas/settings.py:30]: Set back to False for live deployment

**Review by genghisken — CHANGES_REQUESTED** [review]

**Pull request merged**